docs/application/dotnet/guides/nui/text.md

```diff
@@ -374,17 +323,48 @@ label.Text = "&#9786; &#x263a;"; //smile-face
 
 ```
 
+
+### Use Decorations for TextLabel

```

Comment:
As the "Use Decorations for TextLabel" title is needed above, how about changing this title to "TextLabel properties"?